packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -204,8 +227,25 @@ export class TdsTableBodyRowExpandable {
           }}
           part="expand-row"
         >
-          <td class="tds-table__cell-expand" colSpan={this.columnsNumber}>
-            <slot name="expand-row" />
+          <td
+            class={isFlex ? 'tds-table__cell-expandflex' : 'tds-table__cell-expand'}
```

Comment:
```suggestion
            class={isFlex ? 'tds-table__cell-expand-flex' : 'tds-table__cell-expand'}
```

Reply:
Good call! But due to Martins suggestion, I have updated the code in this PR hence this is not applicable no more.